nodeup/pkg/model/context.go

```diff
@@ -313,8 +313,8 @@ func (c *NodeupModelContext) KubectlPath() string {
 
 // BuildCertificatePairTask creates the tasks to pull down the certificate and private key
 func (c *NodeupModelContext) BuildCertificatePairTask(ctx *fi.ModelBuilderContext, key, path, filename string) error {
-	certificateName := fmt.Sprintf("%s/%s.pem", strings.TrimSuffix(path, "/"), filename)
-	keyName := fmt.Sprintf("%s/%s-key.pem", strings.TrimSuffix(path, "/"), filename)
+	certificateName := strings.Join([]string{path, filename + ".pem"}, "/")
```

Comment:
I actually meant `filepath.Join` https://golang.org/pkg/path/filepath/#Join but this works...

Reply:
aahh .. didn't know there was a `filepath.Join` .. actually that's better as does Clean() for us .. no need to Trim